discord/ext/pages/pagination.py

```diff
@@ -1035,8 +1098,8 @@ async def edit(
 
         Parameters
         ----------
-        message: :class:`discord.Message`
-            The message to edit with the paginator.
+        target: Union[:class:`~discord.PartialMessage`, :class:`~discord.Message`, :class:`~discord.Interaction`, :class:`~discord.ApplicationContext`, :class:`~discord.ext.bridge.BridgeContext`]
```

Comment:
Should this be `discord.ext.bridge.BridgeContext` or `discord.ext.bridge.Context`? Or even `discord.ext.bridge. BridgeApplicationContext` and `discord.ext.bridge. BridgeExtContext` individually?